components/modal/modal.js

```diff
@@ -0,0 +1,25 @@
+const modalBtn = document.querySelector('.btn');
+
+const backdrop = document.querySelector('.modal-backdrop');
+const modal = document.querySelector('.modal');
+
+const acceptBtn = document.querySelector('#accept');
+const declineBtn = document.querySelector('#decline');
+
+modalBtn.addEventListener('click', () => {
+  console.log('amar');
```

Comment:
should remove console logs when pushing the code

Reply:
Done!